recipes/eigency/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "eigency" %}
+{% set version = "2.0.0a1" %}
+{% set eigenversion = "3.2.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ eigenversion }}.{{ version }}
+
+source:
+  url: https://github.com/wouterboomsma/eigency/archive/refs/tags/2.0.0a1.tar.gz
+  sha256: 4c9a16adf1710ac647670a8a7516cca8e3f33201fedec9056268df1514b9552f
+
+build:
+  script: EIGEN_INC=${PREFIX}/include/eigen3 {{ PYTHON }} -m pip install --no-deps . -vv  # [not win]
+  script: set EIGEN_INC=%LIBRARY_INC%\eigen3 & {{ PYTHON }} -m pip install --no-deps . -vv  # [win]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - cython
+    - numpy {{ numpy }}
+    - eigen {{ eigenversion }}
+  run:
+    - python
+    - numpy {{ numpy }}
+    - setuptools
```

Comment:
My understanding is the eigency provides a Cython API wrapper around Eigen, so doesn't that mean the end user requires both cython and eigen installed to use it?

Reply:
Cython is needed only during the compilation (it writes c++ from python) so ones the package is generated cython is not needed any more.
Eigen is a header only library, so only needed at compilation. 

This means this package is intended for the developing of other packages.

In my case, I'm working on a end user package (python package accelerated by c++) and I use cython and eigency to pass information efficiently. And eigency is the only dependency that is not present on conda-forge at this moment.